tests/unit/opensearch_utils/test_client.py

```diff
@@ -0,0 +1,186 @@
+import unittest
+from sds_in_a_box.SDSCode.opensearch_utils.action import Action
+from sds_in_a_box.SDSCode.opensearch_utils.index import Index
+from sds_in_a_box.SDSCode.opensearch_utils.document import Document
+from sds_in_a_box.SDSCode.opensearch_utils.payload import Payload
+from sds_in_a_box.SDSCode.opensearch_utils.client import Client
+from opensearchpy import OpenSearch, RequestsHttpConnection, AWSV4SignerAuth
+import boto3
+from botocore.exceptions import ClientError
+
+class TestClient(unittest.TestCase):
+    """tests for client.py"""
+
+    def setUp(self):
+        #Opensearch client Params
+        host = 'search-sds-metadata-uum2vnbdbqbnh7qnbde6t74xim.us-west-2.es.amazonaws.com'
```

Comment:
I wonder if there is a way to make this not hard-coded? This is probably best saved for another ticket, but maybe we should save the output of of the CDK stack somewhere, and then have all the code reference that "somewhere".  Like an output file, environment variables, a small dynamodb database, etc

Reply:
I agree, that would be good to do. I can make another ticket for that.